contracts/SponsorFaucet.sol

```diff
@@ -131,95 +97,80 @@ contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard, WhitelistedRole {
 
     //set bounds for sponsorship
     function setBounds(
-        uint256 gasTotalLimit,
-        uint256 collateralTotalLimit,
-        uint256 gasBound,
-        uint256 collateralBound,
-        uint256 upperBound
+        bounds memory smallBounds
```

Comment:
How about to rename the function to `setSmallBounds`?

Reply:
rm that and use **setBounds** for all